index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
[ ] As stated in the project requirement Kindly Make sure that the your A .css file that includes **one class** with styles for your header (make it in your favorite color) ,

![Capture](https://user-images.githubusercontent.com/93598189/218444684-65cd0968-78a3-4f79-816d-8d1c02289fdc.PNG)

Reply:
Thank you very much for your reviews. I'm making the changes right away.